src/EventSubscriber/EventSubscriber.php

```diff
@@ -120,6 +121,7 @@ public function onKernelRequest(RequestEvent $event) {
 
     if (
       $event->getRequestType() == HttpKernelInterface::MAIN_REQUEST &&
+      (Settings::get('stanford_capture_ownership', FALSE) || getenv('GITHUB_ACTIONS')) &&
```

Comment:
no, just edit the unit test as I mentioned in the other comment.

Reply:
Is there a way to change the `Settings` singleton from within the test?  Because otherwise that's always going to evaluate to FALSE ...

Reply:
https://github.com/SU-SWS/stanford_profile/pull/746#discussion_r1394643528